docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    depends_on:
+      - mongodb
+    build: .
+    restart: unless-stopped
+    environment:
+      # You can leave this as-is.
+      - DB_HOST=mongodb
+      - DB_USER=mongouser
+      - DB_PASSWORD=mongopasword
+      - DB_NAME=xyter
+      - DB_PORT=27017
+      # You have to set these values!
+      - DISCORD_TOKEN=YOUR_DISCORD_TOKEN
+      - DISCORD_CLIENT_ID=YOUR_CLIENT_ID
+      - DISCORD_DEV_GUILD_ID=YOUR_DEV_GUILD_ID
+      - HOSTER_NAME=YOUR_HOSTER_NAME
+      - NODE_ENV=development
+    stdin_open: true
+    tty: true
+  mongodb:
+    image: mongo:latest
```

Comment:
Could you pin mongo to the latest pinned version? [This one](https://hub.docker.com/layers/mongo/library/mongo/4.2.21/images/sha256-68f432c24a2086e234f8cfaa531963c7fcba946f96d60af2f44ac99477f0f780?context=explore) is the latest one right now.